templates/taxi/car_list.html

```diff
@@ -1,17 +1,37 @@
 {% extends "base.html" %}
 
 {% block content %}
-  <h1>Car list</h1>
+  <h1>Car list <a style="text-decoration: none" href="{% url 'taxi:car-create' %}">+</a></h1>
   {% if car_list %}
-    <ul>
+    <table class="table">
+      <tr>
+        <th>ID</th>
+        <th>Model</th>
+        <th>Manufacturer</th>
+        <th></th>
+        <th></th>
+      </tr>
       {% for car in car_list %}
-        <li>
-          <a href="{% url "taxi:car-detail" pk=car.id %} ">{{ car.id }}</a>
-          {{ car.model }} ({{ car.manufacturer.name }})
-        </li>
+        <tr>
+          <td>
+            <a href="{% url "taxi:car-detail" pk=car.id %}">{{ car.id }}</a>
+          </td>
+          <td>
+            {{ car.model }}
+          </td>
+          <td>
+            {{ car.manufacturer.name }}
+          </td>
+          <td>
+            <a style="text-decoration: none" class="btn btn-primary" href="{% url 'taxi:car-update' car.id %}">Update</a>
+          </td>
+          <td>
+            <a style="text-decoration: none" class="btn btn-danger" href="{% url 'taxi:car-delete' car.id %}">Delete</a>
```

Comment:
The {% url %} template tag generates URLs dynamically using the URL name defined in your urls.py file. The parameters you pass must match the named parameters expected by the URL pattern. 

Example:

`<a href="{% url 'taxi:car-update' car.id %}" class="btn btn-primary btn-sm">Update</a>`

> Since your URL pattern uses <int:pk>, you should use pk= in the template tag:

`<a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-primary btn-sm">Update</a>`

Ensure that the href in your template matches the URL pattern, or update the URL pattern to match the parameters used in your template.